src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -181,9 +181,13 @@ A normal toric variety
 ```
 """
 function NormalToricVariety(P::Polyhedron)
-    fan = normal_fan(P)
-    variety = NormalToricVariety(fan)
+    #construct the variety
+    variety = NormalToricVariety(normal_fan(P))
+    
+    # set attributes
     set_attribute!(variety, :polyhedron, P)
+    
+    # return
     return variety
```

Comment:
```suggestion
    variety = NormalToricVariety(normal_fan(P))
    set_attribute!(variety, :polyhedron, P)
    return variety
```
To me it is a bit unclear why the comments are needed. E.g. the comment `# return` when the next line literally starts with `return`. Same for `# set attributes` when the next line literally calls `set_attribute!`. I suggest leaving out comments as much as possible as these are not tested and once the code is changed it often happens that old comments are left and leave to confusion. At least this is what I took from the chapter on comments in the clean code book.

Reply:
I'd phrase it a bit different: comments should not state what you are doing (to see that, one can read the code after all -- or if the code is so opaque that one needs explanations, one should perhaps revise). But definitely have comments that explain *why* you are doing certain things; comments should help illuminate the bigger picture. But of course that's just rule of thumb, too.

In any case, I agree these comments here are redundant

Reply:
You are both right. I will adjust this (hopefully very soon).